.github/workflows/webpack.yml

```diff
@@ -0,0 +1,30 @@
+name: NodeJS with Webpack
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [12.x]
```

Comment:
I have 12.x on my dev box. The original template was 12.x, 14.x and 16.x. The first two seemed to be progressing, but 16.x failed so caused the first two to be cancelled. Let me try adding them back because I didn't retry, but in the end I should be able to add 14.x back at the very least. I had originally created this branch via webpage and I am having trouble using this so I am going to abandon and re-submit a new PR with the changes.